deploy/k8s.yaml

```diff
@@ -0,0 +1,74 @@
+---
+# based on example https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/manifests/node-exporter-daemonset.yaml
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  name: ethtool-exporter
+  namespace: monitoring
+  labels:
+    app: ethtool-exporter
+spec:
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 100%
+  selector:
+    matchLabels:
+      app: ethtool-exporter
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9417'      
+      labels:
+        app: ethtool-exporter
+    spec:
+      hostNetwork: true
+      imagePullSecrets:
+        {{IMAGE_PULL_SECRETS}}
+
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: ethtool-exporter
+        env:
+        - name: IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP      
+        image: {{DOCKER_REGISTRY}}/ethtool-exporter:{{DOCKER_TAG}}
+#        image: drdivano/ethtool-exporter@sha256:b0d883597b5a753d40f5249e1b5e247a49b32b703ae81dec01408d3a47f2912b
+        command:
+          - "sh"
+          - "-exc"
+          - "python3 /ethtool-exporter.py -l $(IP):9417 -I '(eth|em|eno)[0-9]+'"
```

Comment:
Add at least ens and enp. See https://en.wikipedia.org/wiki/Consistent_Network_Device_Naming for details - enp is a bit tricky, i.e. enp0s1.